src/cpp/StatisticsBackend.cpp

```diff
@@ -302,12 +305,13 @@ std::vector<StatisticsData> StatisticsBackend::get_data(
     check_entity_kinds(allowed_kind, entity_ids, db, "Wrong entity id passed in entity_ids");
 
     std::vector<StatisticsData> ret_val;
+    auto t_to_select = t_to - std::chrono::nanoseconds(1);
```

Comment:
will not work in windows because  
```c++
using Timestamp = std::chrono::time_point<std::chrono::system_clock>;
```
and
```c++ 
auto t_to_select = t_to - std::chrono::nanoseconds(1);
```
cannot be done on windows because the system_clock resolution are 100 ns nor 1 ns.
The easy workaround is doing:
```c++
auto t_to_select = t_to - std::chrono::system_clock::duration(100);
```